content/getting-started/sections/why-atom.md

```diff
@@ -3,7 +3,26 @@ title: Why Atom?
 ---
 ### Why Atom?
 
+####  About ATOM  
+   
+ATOM is essentially GitHub’s hackable text editor. It is a free open-source text editor and source code editor for macOS, Linux, and Microsoft Windows with support for plug-ins written in Node.js, and embedded Git Control, developed by GitHub. ATOM is based on Electron, an open source library developed by GitHub for building cross-platform desktop applications with HTML, CSS, and JavaScript.
```

Comment:
> ATOM is essentially GitHub’s hackable text editor.

Remove the word "essentially". I'd also remove the word "GitHub's". Neither word adds additional meaning ("essentially" what? Sure, it's nice to know the folks behind GitHub are behind Atom, but it's a standalone product/app, so this is irrelevant information.)

Reply:
> It is a free open-source text editor and source code editor for macOS, Linux, and Microsoft Windows with support for plug-ins written in Node.js, and embedded Git Control, developed by GitHub.

I'd suggest breaking this into two sentences, as this is a run-on sentence which is trying to tackle two distinct ideas. Try the following:

> Atom is a free, open-source text editor and source code editor for macOS, Linux and Microsoft Windows. It supports plug-ins written in Node.js and embedded Git control.